env/includes/unreal/settings/tk-multi-publish2.yml

```diff
@@ -0,0 +1,72 @@
+# This file is based on templates provided and copyrighted by Autodesk, Inc.
+# This file has been modified by Epic Games, Inc. and is subject to the license
+# file included in this repository.
+
+################################################################################
+
+publish_help_url: &help_url "https://support.shotgunsoftware.com/hc/en-us/articles/115000068574-Integrations-User-Guide#The%20Publisher"
+
+includes:
+- ../../app_locations.yml
+
+settings.tk-multi-publish2.unreal.project:
+  collector: "{self}/collector.py:{engine}/tk-multi-publish2/basic/collector.py"
+  publish_plugins:
+  - name: Publish to ShotGrid
+    hook: "{self}/publish_file.py"
+    settings: {}
+  - name: Upload for review
+    hook: "{self}/upload_version.py"
+    settings: {}
+  - name: Publish to ShotGrid
+    hook: "{self}/publish_file.py:{engine}/tk-multi-publish2/basic/publish_session.py"
+    settings: {}
+  - name: Export FBX and Publish to ShotGrid
+    hook: "{self}/publish_file.py:{engine}/tk-multi-publish2/basic/publish_asset.py"
+    settings:
+        Publish Template: unreal.asset_publish
+  - name: Render Movie and Submit for Review
+    hook: "{self}/publish_file.py:{engine}/tk-multi-publish2/basic/publish_movie.py"
+    settings:
+        Publish Template: unreal.movie_publish
+  help_url: *help_url
+  location: "@apps.tk-multi-publish2.location"
+
+settings.tk-multi-publish2.maya.asset_step.unreal:
```

Comment:
Just want to double check this one as it references `maya` throughout.

Reply:
yes it is: this is to avoid conflicts with the default2 setting. This is the `asset_step` setting for Maya coming from the Unreal integration.